src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx

```diff
@@ -249,9 +249,10 @@ To uninstall and delete the Kubernetes agent operator, run this command:
 >
     Yes, you just need to change your [license key](/docs/apis/intro-apis/new-relic-api-keys/#license-key) injected in the secret in the appropriate namespace.
 
-    <Callout title="PUBLIC PREVIEW">
+    <Callout variant="important">
```

Comment:
@nbaenam I think you may still want the `title="PREVIEW"` in addition to the variant

Reply:
@paperclypse, no, @mangulonr wanted to change this callout.